apps/script/aichatCloudWatchDashboard/constants.js

```diff
@@ -0,0 +1,217 @@
+const DASHBOARD_NAME = 'GenAI';
+
+const modelNames = [
+  'gen-ai-mistral-7b-inst-v01',
+  'gen-ai-arithmo2-mistral-7b',
+  'gen-ai-biomistral-7b',
+  'gen-ai-karen-creative-mistral-7b',
+  'gen-ai-mistral-pirate-7b',
+];
+
+const headerWidgetConfig = {
+  height: 2,
+  width: 24,
+  y: 0,
+  x: 0,
+  type: 'text',
+  properties: {
+    markdown:
+      '# Model Metrics\n*[Metrics Descriptions](https://docs.aws.amazon.com/sagemaker/latest/dg/monitoring-cloudwatch.html#cloudwatch-metrics-jobs)*',
+  },
+};
+
+// Note that the second widget here still has some hard coded model names in it.
+// This could probably be adjusted to accommodate parameterized model names in the future.
```

Comment:
could we generate this config programmatically? looks like a lot of the `SEARCH` queries are pretty similar aside from model name?

Reply:
Yeah I think we definitely can. I was imagining that a) the most important thing is going to be having the base model visible in this figure (which it is with this config), and b) that this dashboard probably isn't perfect and this is the sort of thing we can we can iterate on once we have real data.